global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.6.20310.4",
+    "version": "5.0.100-preview.8.20362.3",
```

Comment:
Right, this is a breaking change that shouldn't have been made outside the monthly Infrastructure Rollout window.

Reply:
@marek-safar can you please send an email to the runtime-repo alias explaining why the SDK's min version was bumped and why it happened outside the batched rollout window? Thanks